terraform/rds.tf

```diff
@@ -8,7 +8,7 @@ resource "aws_rds_cluster" "ehr_out_service" {
   backup_retention_period = 35
   preferred_backup_window = "06:30-08:00"
   vpc_security_group_ids = [
-    aws_security_group.ehr_out_service_db_sg.id,
+    aws_security_group.ehr_out_service_db_sg.id, # TODO: Remove as part of PRMT-4706
```

Comment:
Just a reminder here, checked the board and there seems to be no description on `PRMT-4706`, could we please add one?